integration-web/pom.xml

```diff
@@ -65,6 +65,11 @@
             <groupId>org.powermock</groupId>
             <artifactId>powermock-module-junit4</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This dependency is not required because it has already been imported on integration-core.